openadapt/app/util.py

```diff
@@ -50,9 +86,21 @@ def on_export(dest):
     ui.notify("Exported data.")
 
 
-def sync_switch(switch, prop):
+def sync_switch(switch: Any, prop: Any) -> None:
+    """Synchronize the value of a switch with a property.
+
+    Args:
+        switch: The switch object.
+        prop: The property object.
+    """
     switch.value = prop.value
 
 
-def set_dark(dark_mode, value):
+def set_dark(dark_mode: Any, value: Any) -> None:
```

Comment:
@0dm can you please clarify the types of these as well?

Reply:
dark_mode is ui.DarkMode / ValueElement
value is bool

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/80a53e0c92855a46739d37d6fe0c5cadbdef1bd0